app/frontend/tests/unit/components/base/BaseDialog.spec.js

```diff
@@ -28,4 +28,16 @@ describe('BaseDialog.vue', () => {
 
     expect(wrapper.text()).toMatch('Continue');
   });
+
+  it('renders with no action buttons and the close X', async () => {
```

Comment:
This description is confusing - I'd recommend `renders with the close button` as this test only checks for that.
As well, we should never take the "default" for granted -> the negated case of `renders without the close button` should be explicitly tested, and that unit test should check that there is no "close" in the string.